pennylane/tape/tapes/jacobian_tape.py

```diff
@@ -488,6 +488,11 @@ def jacobian(self, device, params=None, **options):
         if any([m.return_type is State for m in self.measurements]):
             raise ValueError("The jacobian method does not support circuits that return the state")
 
+        if self.is_sampled:
+            raise qml.QuantumFunctionError(
+                "Circuits that include sampling can not be differentiated."
+            )
+
```

Comment:
Another bug? This seems to be an important error! But this was also the case without tape mode?

Reply:
Yep, this was an exception that I forgot to port to the tape-mode case :) It was there in old QNode.